extensions/amp-story/1.0/progress-bar.js

```diff
@@ -193,6 +436,11 @@ export class ProgressBar {
 
     this.updateProgressByIndex_(segmentIndex, progress);
 
+    if (!this.activeSegmentIndex_) {
```

Comment:
Sorry I missed this in the first review: is there any reason why we want to position the segments for the first time in the first call to `updateProgress()` and not in the `build()`?